test-common/utils.c

```diff
@@ -3002,9 +3002,8 @@ eos_test_client_prepare_volume (EosTestClient *client,
       { "GI_TYPELIB_PATH", libeos_updater_util_path, NULL },
       { "LD_LIBRARY_PATH", new_ld_library_path, NULL },
       { "FLATPAK_SYSTEM_HELPER_ON_SESSION", "1", NULL },
-      /* FIXME: Add back G_DEBUG=fatal-warnings after we can rely on
-       * https://gitlab.gnome.org/GNOME/pygobject/commit/806c5059f989ed1b8bc62e6aa1ef55123ac110de */
-      { "G_DEBUG", "gc-friendly", NULL },
+      { "XDG_CACHE_HOME", g_get_user_cache_dir (), NULL },
```

Comment:
Would this not come from the parent process’ environment through `merge_parent_and_child_env()`?

Reply:
No because the parent process has `XDG_CACHE_HOME=/dev/null` because of `G_TEST_OPTION_ISOLATE_DIRS`

Reply:
Worth a comment in the code.

Reply:
Oh yeah, I had a comment in a previous version, guess I didn't copy it over